controllers/booking.js

```diff
@@ -20,17 +20,30 @@ async function getAllByUserId(req, res) {
 }
 
 // @desc    Get all bookings (by restaurant id)
-// @route   GET /booking/restaurant/:id
-// @access  Private
+// @route   GET /booking/restaurant?startDateTime=2024-02-01T00%3A00%3A00%2B08%3A00&endDateTime=2024-02-29T23%3A59%3A59%2B08%3A00
+// @access  Private (bearer token passed in header/ check if user is an owner of restaurant)
 async function getAllByRestaurantId(req, res) {
```

Comment:
restaurant data hasn't been connected so i haven't been able to test this. to validate later .

Reply:
Yes what I implemented for now is filter logic by dateTime not restaurant ID yet.
Example of response is as attached. 
<img width="1131" alt="Screenshot 2024-01-15 at 9 39 30 AM" src="https://github.com/jx0906/proj3-backend/assets/88537845/c3d67232-b195-4cef-82af-894fb7786fd1">

Reply:
yeah, noted as observed! not an issue for me!